lib/utils.pm

```diff
@@ -867,6 +867,11 @@ sub get_x11_console_tty {
       && !check_var('VIRSH_VMM_FAMILY', 'hyperv')
       && !check_var('VIRSH_VMM_TYPE',   'linux')
       && !get_var('VERSION_LAYERED');
+    # start with Tumbleweed build 20190614, we ship gdm 3.32 which runs on tty2
+    # for both auto-login cases and non auto-login cases
+    if (is_tumbleweed() && check_var('DESKTOP', 'gnome') && get_var('BUILD') ge "20190614") {
```

Comment:
Our tests should always consider openSUSE Tumbleweed as the default product without a version. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You can try to negate the logic check.